vms/platformvm/block/executor/acceptor_test.go

```diff
@@ -113,12 +113,6 @@ func TestAcceptorVisitAtomicBlock(t *testing.T) {
 	)
 	require.NoError(err)
 
-	// Set expected calls on the state.
-	// We should error after [commonAccept] is called.
-	s.EXPECT().SetLastAccepted(blk.ID()).Times(1)
-	s.EXPECT().SetHeight(blk.Height()).Times(1)
-	s.EXPECT().AddStatelessBlock(blk).Times(1)
```

Comment:
assuming these changes are only relevant in the error case, therefore the behavior does not change overall

Reply:
Yeah, this test is now failing sooner than it was before, so we are no longer expecting to hit these calls in this case.

These calls still exist in the happy path where the block acceptance succeeds.